extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/ranger/DeltaCatalogRangerSparkExtensionSuite.scala

```diff
@@ -432,6 +432,25 @@ class DeltaCatalogRangerSparkExtensionSuite extends RangerSparkExtensionSuite {
       }
     }
   }
+
+  test("optimize path-based table") {
+    assume(isSparkV32OrGreater)
```

Comment:
can we explain the reason why 3.1 is unavailable for testing?

Reply:
Optimize is available in Delta Lake 1.2.0 and above.
https://docs.delta.io/latest/optimizations-oss.html#-optimize-performance-with-file-management
https://github.com/delta-io/delta/releases/tag/v1.2.0

Reply:
assume(isSparkV32OrGreater, "hint")

replace the hint with proper comments

Reply:
Done